packages/messenger-widget/src/components/ConfigureProfile/bl.tsx

```diff
@@ -226,21 +220,28 @@ export const namingServices = [
     {
         name: NAME_SERVICES.ENS,
         chainId: 1,
-        component: <ConfigureEnsProfile />,
     },
     {
         name: NAME_SERVICES.GENOME,
         chainId: 100,
-        component: <ConfigureGenomeProfile />,
     },
 ];
 
 export const fetchComponent = (name: string) => {
+    let chain: number = 1;
```

Comment:
remove chain var. Instead it within the context
`
      case NAME_SERVICES.ENS:
            if (process.env.REACT_APP_CHAIN_ID === '5') {
                return <ConfigureEnsProfile chainToConnect={5} />;
            } 
            return <ConfigureEnsProfile chainToConnect={1} />;
      case NAME_SERVICES.GENOME:
            const genomeChainId = namingServices[1].chainId;
            return <ConfigureGenomeProfile chainToConnect={genomeChainId} />;

`